### ISSUE TITLE
Issue running t2i XL models

### ISSUE THREAD

**Issue by Alverik** [Mikubill/sd-webui-controlnet#2059]
I'm on windows 10, NVIDIA GTX970, using Automatic1111 v1.6.0. CN v1.1.403, The checkpoint is KohakuXL alpha7, 512*768, batch size 4, text to image tab. Anyway, whenever I try to use an XL t2i  model I get this error:

```
*** Error running before_process_batch: D:\Program Files\Automatic1111\extensions\sd-webui-controlnet\scripts\controlnet.py
    Traceback (most recent call last):
      File "D:\Program Files\Automatic1111\modules\scripts.py", line 627, in before_process_batch
        script.before_process_batch(p, *script_args, **kwargs)
      File "D:\Program Files\Automatic1111\extensions\sd-webui-controlnet\scripts\controlnet.py", line 972, in before_process_batch
        self.controlnet_main_entry(p)
      File "D:\Program Files\Automatic1111\extensions\sd-webui-controlnet\scripts\controlnet.py", line 688, in controlnet_main_entry
        model_net = Script.load_control_model(p, unet, unit.model)
      File "D:\Program Files\Automatic1111\extensions\sd-webui-controlnet\scripts\controlnet.py", line 321, in load_control_model
        model_net = Script.build_control_model(p, unet, model)
      File "D:\Program Files\Automatic1111\extensions\sd-webui-controlnet\scripts\controlnet.py", line 350, in build_control_model
        network = build_model_by_guess(state_dict, unet, model_path)
      File "D:\Program Files\Automatic1111\extensions\sd-webui-controlnet\scripts\controlnet_model_guess.py", line 180, in build_model_by_guess
        adapter.load_state_dict(state_dict, strict=False)
      File "D:\Program Files\Automatic1111\venv\lib\site-packages\torch\nn\modules\module.py", line 2041, in load_state_dict
        raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
    RuntimeError: Error(s) in loading state_dict for Adapter:
        size mismatch for body.2.in_conv.weight: copying a param with shape torch.Size([1280, 640, 1, 1]) from checkpoint, the shape in current model is torch.Size([640, 320, 1, 1]).
        size mismatch for body.2.in_conv.bias: copying a param with shape torch.Size([1280]) from checkpoint, the shape in current model is torch.Size([640]).
```

I had initially downloaded the models from civitai (https://civitai.com/models/136070), but redownloaded the files from the link you provide just in case, but same issue happens. Other models like Kohya seem to work fine.

**Comment by lllyasviel** [comment]
I tested again and it does not have problem.
If anyone else has the same problem please let us know

**Comment by KohakuBlueleaf** [comment]
off topic sry

I think what you are using is Kohaku-XL alpha7 not beta7(

**Comment by Alverik** [comment]
@lllyasviel I think I fixed it. Let me explain. After reading your comment I tried a few things (I tried to pull the latest version, I tried to revert all the files in the cn folder, and clone it again but it didn't help). Then I went and compared my cn extension folder to the one I have in my SD.NEXT folder to see if there was any difference. I should add I usually put my controlnet models in the webui's "models/controlnet" folder to replace the extension easier. Anyway, I checked and noticed the folder inside the extension in my automatic1111 installation was empty, whereas the same folder in my SD.NEXT had all the default yaml files... I don't bother putting those files in my models/controlnet folder so I didn't notice they were missing... I copied the yaml files to automatic1111's models/controlnet folder then tested again. I found t2i xl models listed as supported don't throw the error anymore. Only the unlisted TencentArc and SargeZT t2i models I got from civitai throw it now. So I guess the problem was the missing yaml files. No idea why they weren't there.

Anyway, I hope the rest of the t2i models are supported soon, I specially want to use that SargeZT's Segmentation model, haha...
